exercises/bowling/tests/bowling.rs

```diff
@@ -0,0 +1,320 @@
+extern crate bowling;
+
+use bowling::*;
+
+#[test]
+fn roll_returns_a_result() {
+    let game = BowlingGame::new();
+    assert!(game.roll(0).is_ok());
+}
+
+#[test]
+fn you_can_not_roll_more_than_ten_pins_in_a_single_roll() {
+    let game = BowlingGame::new();
+
+    assert!(game.roll(11).is_err());
+}
+
+#[test]
+fn a_game_score_is_ok_if_ten_frames_have_been_rolled() {
+    let game = BowlingGame::new();
+
+    for _ in 0..10 {
+        game.roll(0).is_ok();
```

Comment:
for this and others - the bool from `is_ok` is being discarded, and I think this is confusing because it makes the intent unclear. If we're not going to check whether it's OK or not, it seems clearer to say `let _ = game.roll(0);` so that there is no confusion about intent.